### DISCUSSION TITLE
easiest(?) hardware

### DISCUSSION THREAD

**Discussion by marekorok** [sle118/squeezelite-esp32#215] [Show and tell]
Hi,

I had old Yamaha amplifier lying around and was wondering to add Spotify/Bluetooth features, and come up with this simple hardware config for Squezelite-esp32:

1. TTGO T7 V1.5, ESP32-WROVER-E
2. 0.91" 128x32 I2C White OLED, SSD1306 -> on I2C port, connector JST 1mm, config:  SDA 21/SCL 22
3. Optical Fiber Transmitter: Dlt1150/Audio  -> 5V/GND/GPIO15

![IMG_20221209_104115](https://user-images.githubusercontent.com/28598034/206675354-2034f720-514e-4c8d-a447-2ae0b644b2d2.jpg)

Working great for couple of days now.

Thank you for the great project!

PS. if there would be only stream player added for Internet radio then I'm in heaven :-)

**Comment by wizmo2** [comment]
Interested in how you configured the optical output in the NVS.  (?)

And is the DLT1150 wired directly to the output pins?

**Reply by marekorok** [reply]
since I know nothing (yet) about the NVS didn't touch it and just enabled the SPDIF, it is sharing the same output port.

Yes, DLT is wired directly to output pin. no harm
![image](https://user-images.githubusercontent.com/28598034/206745798-606d88ee-40dc-4825-a180-923c633db926.png)

**Comment by dkotte** [comment]
_"PS. if there would be only stream player added for Internet radio then I'm in heaven :-)"_
What do you mean?
I play internet radio from my LMS.
